config/initializers/devise.rb

```diff
@@ -142,7 +142,7 @@
   # initial account confirmation) to be applied. Requires additional unconfirmed_email
   # db field (see migrations). Until confirmed, new email is stored in
   # unconfirmed_email column, and copied to email column on successful confirmation.
-  config.reconfirmable = true
+  config.reconfirmable = false
```

Comment:
ok, me souviens pas avoir mis ça, si c'est moi c'était une erreur.

Reply:
It was enabled by default :)